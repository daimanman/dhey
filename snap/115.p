{"C":1,"Duration":110706868,"EndTime":1551409819141694900,"Err":"","FileId":"","InputFileName":"","LonhApi":"signOnline","Method":"","N":1,"P":null,"Payload":"","QH":{},"Remark":"","ReqNums":1,"StartTime":1551409819030988000,"Status":1,"TaskId":115,"Type":"LONH","Url":"","Z":""}