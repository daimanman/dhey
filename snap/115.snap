{"AvgTotal":0.1007181,"Fastest":0.1007181,"Slowest":0.1007181,"Average":0.1007181,"Rps":9.92870199100261,"AvgConn":0.0488551,"AvgDNS":0.0009847,"AvgReq":0,"AvgRes":0,"AvgDelay":0.051863,"ConnMax":0.0009847,"ConnMin":0.0009847,"DnsMax":0.0009847,"DnsMin":0.0009847,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0,"DelayMax":0.051863,"DelayMin":0.051863,"Lats":[0.1007181],"ConnLats":[0.0009847],"DnsLats":[0.0009847],"ReqLats":[0],"ResLats":[0],"DelayLats":[0.051863],"Total":100718100,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":0,"SizeReq":0,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.1007181,"Count":1,"Frequentcy":1},{"Mark":0.1007181,"Count":0,"Frequentcy":0},{"Mark":0.1007181,"Count":0,"Frequentcy":0},{"Mark":0.1007181,"Count":0,"Frequentcy":0},{"Mark":0.1007181,"Count":0,"Frequentcy":0},{"Mark":0.1007181,"Count":0,"Frequentcy":0},{"Mark":0.1007181,"Count":0,"Frequentcy":0},{"Mark":0.1007181,"Count":0,"Frequentcy":0},{"Mark":0.1007181,"Count":0,"Frequentcy":0},{"Mark":0.1007181,"Count":0,"Frequentcy":0},{"Mark":0.1007181,"Count":0,"Frequentcy":0}],"TaskId":115,"Duration":0.1007181}